Assets/MixedRealityToolkit.Examples/Experimental/SpatialAwareness/DemoSpatialAwarenessSceneObjectConsumer.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using Microsoft.MixedReality.Toolkit.Experimental.SpatialAwareness;
+using Microsoft.MixedReality.Toolkit.Physics;
+using Microsoft.MixedReality.Toolkit.SpatialAwareness;
+using UnityEngine;
+
+public class DemoSpatialAwarenessSceneObjectConsumer : MonoBehaviour, ISpatialAwarenessSceneObjectConsumer
```

Comment:
Please put these in a namespace such as Microsoft.MixedReality.Toolkit.Experimental.Examples.<<YOUR EXPERIMENTAL FEATURE NAME>>

Reply:
Added missing namespace.